tensorboard/plugins/audio/tf_audio_dashboard/tf-audio-dashboard.html

```diff
@@ -113,10 +118,14 @@
         _runToTag: Object,  // map<run: string, tags: string[]>
         _dataNotFound: Boolean,
 
+        _tagFilter: {
+          type: String,  // upward bound from paper-input
+          value: '.*',
```

Comment:
In the current version, the search query is not bound to the URL and is therefore not shared across dashboards. I'm open to changing this. I can see advantages of each way: on the one hand, any given tag should usually belong to just one dashboard (it's not both an image summary and a scalar summary), so allowing separate queries is nice; on the other hand, perhaps you want to filter for just some _kind_ of tag across all dashboards (e.g., if you have `unweighted_precision` and `weighted_precision` scalars, and `unweighted_activations` and `weighted_activations` histograms, then perhaps filtering by `unweighted` or `(?!un)weighted` would be useful).